numpy/ma/core.py

```diff
@@ -5860,15 +5860,16 @@ def __getstate__(self):
 
         """
         cf = 'CF'[self.flags.fnc]
-        state = (1,
-                 self.shape,
-                 self.dtype,
-                 self.flags.fnc,
-                 self._data.tobytes(cf),
-                 # self._data.tolist(),
-                 getmaskarray(self).tobytes(cf),
+        data_state = super(MaskedArray, self).__reduce__()[2]
+        print(data_state)
+        state = (data_state[0],     # version
```

Comment:
Can you just do `data_state + (getmaskarray(self).tobytes(cf), self._fill_value)`?